it/src/main/scala/encry/it/api/HttpApi.scala

```diff
@@ -96,6 +111,45 @@ trait HttpApi { // scalastyle:ignore
     )
   }
 
+  def outputs: Future[Seq[EncryBaseBox]] = get("/wallet/utxos").flatMap { r =>
```

Comment:
As far as I understood, these functions are only used for tests, so, better make another `HttpApiTest` with all these functions and use it in tests